src/Sylius/Bundle/UserBundle/Resources/config/serializer/Model.Customer.yml

```diff
@@ -5,27 +5,35 @@ Sylius\Component\User\Model\Customer:
         id:
             expose: true
             type: integer
+            groups: [Default, Detailed]
         email:
             expose: true
             type: string
+            groups: [Detailed]
         emailCanonical:
             expose: true
             type: string
+            groups: [Default, Detailed]
```

Comment:
I would say that `email` should have `Default` group also and `emailCanonical` only `Detailed`. Exactly the same as you did with `username` and `usernameCanonical`.